recipe/meta.yaml

```diff
@@ -19,22 +22,31 @@ requirements:
     - {{ cdt('libx11-devel') }}          # [linux]
     - {{ cdt('libxcb') }}                # [linux]
     - {{ cdt('libxau') }}                # [linux]
+    - make                               # [linux]
+    - m2-patch                           # [win]
   host:
-    - tk
+    - tk {{ tk }}
```

Comment:
if hardcoding versions in scripts, pin to it

Reply:
As there is a fix in bld.bat without hardcoding versions I leave it here as it is